README.md

```diff
@@ -15,184 +15,45 @@
 
 </div>
 
-Fern is an open-source toolkit that simplifies the design, build, and consumption of REST APIs. It allows you to effortlessly generate SDKs, API documentation, and backend server boilerplate.
+Fern is a platform for generating Stripe-level SDKs (client libraries) and Docs for your API. Get started by passing in your OpenAPI specification.
 
 <div align="center">
     <img src="/fern/docs/images/overview.png" width="700" alt="Overview Diagram">
 </div>
 
-## Getting started
+### 🌿 SDKs (API Clients)
 
-_Note: Fern requires **Node 18** or higher_
+Our generators read in your OpenAPI spec as input and output SDKs. Fern handles publishing the generated SDKs to GitHub and package managers like NPM and PyPI.
```

Comment:
```suggestion
Our generators read in your OpenAPI spec to input and output SDKs. Fern handles publishing the generated SDKs to GitHub and package managers like NPM and PyPI.
```

Reply:
Or _and_ perhaps instead of _as_? At least, _as_ doesn't read right to me. If you go with _and_, perhaps change _input and output_ to _input/output_ so that _and_ isn't repeated twice.